source/includes/administration/_pricings.md

```diff
@@ -231,6 +280,13 @@ Attributes | &nbsp;
 `pricingProducts.product`<br/>*Object* | The product attached to this product pricing element.
 `pricingProducts.cogs`<br/>*double* | The Cost Of Goods Sold (COGS) of the product.
 `pricingProducts.currency`<br/>*string* | The currency of the pricing.
+`pricingProducts.pricingTiers`<br/>*Array[Object]* | The list of pricing tiers assigned to the product pricings.
+`pricingProducts.pricingTier.id`<br/>*UUID* | UUID of the pricing tier.
+`pricingProducts.pricingTier.pricingMode`<br/>*Enum* | The pricing mode of the pricing tier. Must be one of "FLAT_FEE" or "PER_UNIT".
+`pricingProducts.pricingTier.lowerBound`<br/>*double* | The pricing tier's lower bound.
+`pricingProducts.pricingTier.upperBound`<br/>*double* | The pricing tier's upper bound.
+`pricingProducts.pricingTier.price`<br/>*Map[string, double]>* | The currencies of the pricing tier.
```

Comment:
This kinda sounds like you should be providing a list of currencies. Should it be "The price of the pricing tier in each currency" or something like that? Or "The currency based prices of the pricing tier"

Reply:
5a58e93bff8b0d81cebf825582a5ffe310195592